### PR TITLE
feat navigation component added

### PR DESCRIPTION
**Pull request by Himanshu-Dhawale**
Screenshot 10

### CODE REVIEW

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Sharmaryan) [inline components/navigation/navigation.html]: fix : title

File: components/navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        <div class="container_components">
+            <nav class="nav-bar">
+                <div class="nav-title">Title</div>
+                <button class="nav-icon"><i class='bx bxs-user'></i></button>
+                <button class="nav-icon"><i class='bx bxs-heart'></i></button>
+                <button class="nav-icon"><i class='bx bxs-cart'></i></button>
+            </nav>
+        </div>
+        <div class="container-code">
+            <iframe src="https://www.thiscodeworks.com/embed/6210af491cc8b9001542eddd" style="width: 100%; height: 164px;" frameborder="0"></iframe>
```
Comment (Sharmaryan) [inline components/navigation/navigation.html:83]: again I would say don't use inline css

File: components/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                    </a>
+                    <a href="../navigation/navigation.html" class="side_link active">
+                        <span class="side_name">Navigation</span>
+                    </a>
+                    </div>
+        </aside>
+    </div>
+    <!-- sidebar ends -->
+
+    <!-- navbar component starts -->
```
Comment (Sharmaryan) [inline components/navigation/navigation.html:69]: good use of comments

**Review by Sharmaryan — APPROVED** [review]

**Comment by Sharmaryan** [comment]
other than these changes everything looks good

**Review by samyakshah3008 — APPROVED** [review]
Clean code, good to go

**Pull request merged**